app/main/Mindfulness.tsx

```diff
@@ -27,16 +27,17 @@ function Mindfullness() {
   return (
     <div className="mindfulness-container">
       <h2 className="title">Mindfulness</h2>
-      <div className="video-container">
-        {videos.map((video, index) => 
-            generateVideoPlayer({
+      {videos.map((video, index) => (
```

Comment:
Better to key on name rather than index, see:
https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/

Reply:
Completed